fastlane/fastlane/Fastfile

```diff
@@ -121,21 +121,63 @@ end
 
 desc "Makes sure the tests on https://docs.fastlane.tools still work with the latest version"
 lane :verify_docs do
+  local_fastlane_dir = File.expand_path("../..")
+  clone_docs do
+    temporary_gemfile = []
+    Fastlane::TOOLS.each { |tool| temporary_gemfile << "gem '#{tool}', path: '#{File.join(local_fastlane_dir, tool.to_s)}'" }
+    temporary_gemfile = temporary_gemfile.join("\n")
+    puts `sed -i -e "s/activate_bin_path/bin_path/g" $(which bundle)` # workaround for bundler https://github.com/bundler/bundler/issues/4602#issuecomment-233619696
+    sh "bundle install"
+    sh "pip install -r requirements.txt --ignore-installed" if Helper.ci?
+    sh "bundle exec fastlane test"
+  end
+end
+
+desc "Update the actions.md on https://docs.fastlane.tools"
+desc "This will also automatically submit a pull request to fastlane/docs"
+lane :update_docs do
+  clone_docs do
+    require 'fastlane/documentation/markdown_docs_generator'
+    actions_md_path = File.expand_path("docs/actions.md")
+    Fastlane::MarkdownDocsGenerator.new.generate!(target_path: actions_md_path)
+
+    # Create a new branch
+    sh "git checkout -b 'update-actions-md'"
+
+    Dir.chdir("fastlane") do # this is an assumption of fastlane, that we have to .. when shelling out
+      if `git status -s`.length > 0
```

Comment:
Check out `clone_docs`, it will clone fastlane/docs in a temporary directory, so we don't have to care about what state we leave the directory afterwards, since we delete it anyway